public/js/deckbuilder.js

```diff
@@ -172,33 +172,47 @@ function renderDraft() {
 
   var lands = 0;
   var cards = 0;
+  var creatures = 0;
   //fill up deck
   deck.playerdeck.forEach(function(col, index) {
     var colhtml = "";
+    var i = 0;
```

Comment:
You don't have to define a new variable here; you can just use `col.length` everywhere you are using `i`.